### PR TITLE
Upgrade to Laravel 9

### PR DESCRIPTION
**Pull request by hmennen90**
Tests are ok - only updated composer dependencies

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,10 +10,10 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
-        "illuminate/console": "^5.5|^6.0|^7.0|^8.0",
-        "illuminate/contracts": "^5.5|^6.0|^7.0|^8.0",
-        "illuminate/support": "^5.5|^6.0|^7.0|^8.0",
+        "php": "^8.0",
```
Comment (ilvalerione) [inline composer.json:13]: To keep compatibility with older versions it must remain ">=7.2"

**Review by ilvalerione — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (hmennen90) [reply composer.json]: Is added now

**Comment by ilvalerione** [comment]
Thank you @hmennen90 for the contribution! 👍👍

**Comment by hmennen90** [comment]
@ilvalerione you're welcome :-) if needed or possible  - always

**Pull request merged**